lib/logic.rb

```diff
@@ -0,0 +1,106 @@
+class Game
+  attr_accessor :real_grid
+  def initialize()
+    @real_grid = Array.new(3, 0) { Array.new(3, 0) }
+    p @real_grid[0][1]
+    k = 1
+    3.times do |i|
+      3.times do |j|
+        @real_grid[i][j] = k.to_s
+        k += 1
+      end
+    end
+  end
+
+  def check_winner(board)
+    get_realG(board)
```

Comment:
Naming convention for ruby is the snake_case not camelCase. So get_realG should be get_real_g. Again we stick to conventions because we must consider 'collaboration' with others.